### PR TITLE
feat(navbar): add navbar with test

### LINKED ISSUE #2: Design Navbar
**Task title**
Design Navbar 

**Task description**
Design Navbar component as shown in screenshots, responsive and (ar, en) layouts and  (unSigned, Signed-in) layouts.

**Subtasks**

1. Logo.
2. Navigation.
3. Language Selector.
4. User Avatar.
5. Sign Up button.
6. Sign In button.
7. User drop Down List.

**Screenshots**
Large Screen
Header
unSignHeader

Tablet Screen
header(Tablet)

HeaderUnSigned(tablet)

Header(tablet User Version)

**Link to the component on Figma**
[NavBar](https://www.figma.com/file/IEhYhDLnrHmUuYercEZUjA/Pebble-work?node-id=1830-8562&t=gl9EKVkeewmOfhRc-4)

### PR DESCRIPTION
**Pull request by RuqaiahSaif**
create a responsive navbar with a test

## screenshot
nav
nav1

### CODE REVIEW

File: postcss.config.js
Comment (ramizapk) [inline postcss.config.js:1]: Great job

**Review by ramizapk — APPROVED** [review]
Great job

**Review by fatGubari — COMMENTED** [review]
I love how your code organized, but don't forget to add comments in your code.

**Review by fatGubari — APPROVED** [review]
amazing

**Pull request merged**